content/goldcard-holders-faq/life-in-taiwan.md

```diff
@@ -27,7 +27,7 @@ government agencies invite us to parties or ask our opinions on certain matters.
 
 ## What happens if I change my address?
 You must log in to the Gold Card Application portal and update your details there within 14 days of
- moving.
+ moving. It will cost NTD 500, that can get refunded.
```

Comment:
Suggested wording tweak: "Address changes are free, however the system can't support free changes at the moment. You will be initially charged 500NT, but within a few weeks you will receive communication about your refund." .... though that might be too verbose.